pixl_dcmd/src/pixl_dcmd/main.py

```diff
@@ -63,12 +60,29 @@ def should_exclude_series(dataset: Dataset) -> bool:
     return False
 
 
-def anonymise_and_validate_dicom(dataset: Dataset) -> dict:
+def anonymise_and_validate_dicom(
```

Comment:
perhaps explicitly set `synchronise_pixl_db=True` here:
https://github.com/UCLH-Foundry/PIXL/blob/5f8b3074c2ae26d542610e558a83c41aceb11186/orthanc/orthanc-anon/plugin/pixl.py#L253
just in case the default ever changes